horovod/mxnet/tensor_util.cc

```diff
@@ -123,14 +123,14 @@ int TensorUtil::GetDevice(NDArray* tensor) {
 // If dev_id equal to CPU_DEVICE_ID, construct Tensor on CPU
 // Otherwise construct on GPU
 NDArray* TensorUtil::New(int device, int dtype) {
+  int dev_type = gpu::kDevMask;
```

Comment:
Wouldn't it be cleaner to use a ternary operator here as:
```suggestion
  int dev_type = (device == CPU_DEVICE_ID) ? cpu::kDevMask: gpu::kDevMask;
```

Reply:
I also need to set the device to 0 if it is CPU. I will need to have two such ternary operator statements, which does not differ much with the current one.